src/index.tsx

```diff
@@ -39,6 +40,7 @@ function Updaters() {
       <UserUpdater />
       <ApplicationUpdater />
       <TransactionUpdater />
+      <BlockUpdater />
       <MulticallUpdater />
```

Comment:
Do I need to include a MulticallUpdater in the widget?

Reply:
> you have to render it or you won't get any data

https://uniswapteam.slack.com/archives/C023ZR8Q1DE/p1641946622034500?thread_ts=1641946408.032600&cid=C023ZR8Q1DE